lib/Doctrine/ORM/Tools/SchemaTool.php

```diff
@@ -149,7 +149,8 @@ public function getSchemaFromMetadata(array $classes)
         $metadataSchemaConfig   = $schemaManager->createSchemaConfig();
 
         $metadataSchemaConfig->setExplicitForeignKeyIndexes(false);
-        $schema = new Schema([], [], $metadataSchemaConfig);
+        $namespaces = ! $withNamespaces ? [] : $this->getPlatformNamespaces();
```

Comment:
I think 
`$namespaces = $withNamespaces ? $this->getPlatformNamespaces() : [];`
would be more clear to read.